<!doctype html><html lang=en><head><meta name=generator content="Hugo 0.97.0"><meta charset=utf-8><meta http-equiv=x-ua-compatible content="ie=edge"><title>Home - BLAST Working Group</title><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon type=image/png href=/favicon.png><link rel=preconnect href=https://fonts.googleapis.com><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700;800&display=swap" rel=stylesheet><link href="https://fonts.googleapis.com/css?family=PT+Serif:400,700" ; rel=stylesheet><link href="https://fonts.googleapis.com/css2?family=Nanum+Myeongjo" ; rel=stylesheet><link rel=stylesheet href=/css/style.min.44d270fd44b410cd7d8861ed39062788da870403b08c8580287aecbda92fc0fc.css><meta name=description content="JHU Blast Working Group Website"><meta property="og:title" content="Home"><meta property="og:type" content="website"><meta property="og:url" content="https://jhublast.github.io/"><meta property="og:description" content="JHU Blast Working Group Website"><meta name=twitter:card content="summary"></head><body class="page page-home"><div id=main-menu-mobile class=main-menu-mobile><ul><li class=menu-item-home><a href=/><span>Home</span></a></li><li class=menu-item-research><a href=/research/><span>Research</span></a></li><li class=menu-item-people><a href=/people/><span>People</span></a></li><li class=menu-item-events><a href=/events/><span>Events</span></a></li></ul></div><div class=wrapper><div class=header><div class=container><div class=row><div class=col-12><div class=header-inner><h3 class=header-title><a href=https://jhublast.github.io/>BLAST Working Group</a></h3></div></div></div><div id=main-menu class=main-menu><ul><li class="menu-item-home active"><a href=/><span>Home</span></a></li><li class=menu-item-research><a href=/research/><span>Research</span></a></li><li class=menu-item-people><a href=/people/><span>People</span></a></li><li class=menu-item-events><a href=/events/><span>Events</span></a></li></ul></div><button id=toggle-main-menu-mobile class="hamburger hamburger--slider" type=button aria-label="Mobile Menu">
<span class=hamburger-box><span class=hamburger-inner></span></span></button></div></div><div class=intro><div class=container><div class="row justify-content-start"><div class="col-12 col-md-7 col-lg-6 order-2 order-md-1"><h1 id=jhu-blast-working-group>JHU BLAST Working Group</h1><p><strong>B</strong>ayesian <strong>L</strong>earning <strong>a</strong>nd <strong>S</strong>patio-<strong>T</strong>emporal modeling<br><a href=https://publichealth.jhu.edu/departments/biostatistics>Department of Biostatistics</a><br><a href=https://publichealth.jhu.edu/>Johns Hopkins Bloomberg School of Public Health</a></p></div><div class="col-12 col-md-5 col-lg-6 order-1 order-md-2 position-relative"><img alt class="intro-image intro-image-absolute intro-image-hide-mobile" src=/images/cover.png></div></div></div></div><div class=strip><div class="container pt-6 pb-4 pb-md-4"><div class=row><div class=col-12><h1 style=font-size:2.5rem class=mb>Leadership</h1></div></div><div class="row justify-content-start"><div class="col-12 col-md-6 mb-4"><div class="summary summary-large"><div class=image><a href=https://abhidatta.com/><img alt="Photo of Abhi Datta" class="img-fluid mb-2" src=/images/people/abhi_datta.jpg></a></div><div class=meta><h2 class=team-name><a href=https://abhidatta.com/>Abhi Datta</a></h2><p class=title>Associate Professor</p><p class=title></p><p class=department>Department of Biostatistics</p></div><div class=content><p>Abhi develops statistical and machine learning methods for large spatial datasets as well as Bayesian models for multi-source epidemiological datasets.</p></div></div></div><div class="col-12 col-md-6 mb-4"><div class="summary summary-large"><div class=image><a href=https://aki-nishimura.github.io/><img alt="Photo of Aki Nishimura" class="img-fluid mb-2" src=/images/people/aki_nishimura.jpg></a></div><div class=meta><h2 class=team-name><a href=https://aki-nishimura.github.io/>Aki Nishimura</a></h2><p class=title>Assistant Professor</p><p class=title></p><p class=department>Department of Biostatistics</p></div><div class=content><p>Aki uses Bayesian methods and statistical computing to tackle methodological challenges in healthcare analytics and large-scale biomedical applications.</p></div></div></div></div><div class=people-box><center><a href=people class=button>See all members</a></div></div></div><div class=strip><div class="container pt-0 pb-4 pb-md-4"><div class=row><div class=col-12><h1 style=font-size:2.5rem class=mb>Upcoming Events</h1></div></div><div class="row justify-content-start"><div class="col-12 col-md-12 mb-4"><div class="summary summary-large"><div class=meta><h2 class=event-name>nnSVG: scalable identification of spatially variable genes using nearest-neighbor Gaussian processes</h2><p>Wednesday, April 20, 2022</p><p>Speaker: <a href=https://lmweber.org/>Dr. Lukas Weber</a></p></div><div class=content><p>Spatially-resolved transcriptomics enables the measurement of transcriptome-wide gene expression along with spatial coordinates of the measurements within tissue samples. Depending on the technological platform, this is achieved either by tagging messenger RNA (mRNA) molecules with spatial barcodes followed by sequencing, or through fluorescence imaging-based in-situ transcriptomics techniques where mRNA molecules are detected along with their spatial coordinates using sequential rounds of fluorescent barcoding. During computational analyses of spatially-resolved transcriptomics data, an important initial analysis step is to apply feature selection methods to identify a set of genes that vary in expression across the tissue sample of interest. These genes are referred to as &lsquo;spatially variable genes&rsquo; (SVGs). These SVGs can then be further investigated individually as potential markers of biological processes, or used as the input for further downstream analyses such as spatially-aware unsupervised clustering of cell populations. Here, we propose &rsquo;nnSVG&rsquo;, a new scalable approach to identify SVGs based on nearest-neighbor Gaussian processes (NNGPs), which applies NNGPs in the context of spatially-resolved transcriptomics data. Our method identifies SVGs with flexible length scales per gene, optionally within spatial domains (subregions of the tissue slide), and scales linearly with the number of spatial locations. The linear computational scalability ensures that the method can be applied to the latest technological platforms with thousands or more spatial locations per tissue slide. We demonstrate the performance of our method using experimental data from several technological platforms and simulations, and show that it outperforms existing approaches. A software implementation is available from Bioconductor and GitHub.</p></div></div></div><div class="col-12 col-md-12 mb-4"><div class="summary summary-large"><div class=meta><h2 class=event-name>Bypassing Markov Chains for Bayesian Generalized Linear Mixed Effects Models</h2><p>Wednesday, May 4, 2022</p><p>Speaker: <a href=https://sites.google.com/site/jonathanbradley28/>Dr. Jonathan R. Bradley</a></p></div><div class=content><p>Markov chain Monte Carlo (MCMC) is an all-purpose tool that allows one to generate dependent replicates from a posterior distribution for effectively any Bayesian hierarchical model. As such, MCMC has become a standard in Bayesian statistics. However, convergence issues, tuning, and the effective sample size of the MCMC are nontrivial considerations that are often overlooked or can be difficult to assess. Moreover, these practical issues can produce a significant computational burden. This motivates us to consider finding closed-form expressions of the posterior distribution that are computationally straightforward to sample from directly. We focus on a broad class of Bayesian generalized linear mixed-effects models (GLMM) that allows one to jointly model data of different types (e.g., Gaussian, Poisson, and binomial distributed observations). Exact sampling from the posterior distribution for Bayesian GLMMs is such a difficult problem that it is now arguably overlooked as a possible problem to solve. To solve this problem, we derive a new class of distributions that gives one the flexibility to specify the prior on fixed and random effects to be any conjugate multivariate distribution. We refer to this new distribution as the generalized conjugate multivariate (GCM) distribution. The expression of the exact posterior distribution is given along with the steps to obtain direct independent simulations from the posterior distribution. These direct simulations have an efficient projection/regression form, and hence, we refer to our method as Exact Posterior Regression (EPR). Several illustrations are provided.</p></div></div></div></div><div class=events-box><center><a href=events class=button>See all events</a></center></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/medium-zoom/1.0.6/medium-zoom.min.js integrity="sha512-N9IJRoc3LaP3NDoiGkcPa4gG94kapGpaA5Zq9/Dr04uf5TbLFU5q0o8AbRhLKUUlp8QFS2u7S+Yti0U7QtuZvQ==" crossorigin=anonymous referrerpolicy=no-referrer></script>
<script>const images=Array.from(document.querySelectorAll("img.research-image"));images.forEach(e=>{mediumZoom(e,{margin:0,scrollOffset:40,container:null,template:null})})</script><div class=footer><div class=container><div class=row><div class=col-12><div class=footer-inner><h3 class=footer-title>BLAST Working Group</h3><div id=footer-menu class=footer-menu><ul></ul></div></div></div></div></div></div><script type=text/javascript src=/js/scripts.min.8504133605a277da18f0d58cfd2e90d154962f4a961543a6e2f0a459a2d05462.js></script></body></html>