<!doctype html><html lang=en><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="ie=edge"><title>Events - BLAST Working Group</title><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon type=image/png href=/favicon.png><link rel=preconnect href=https://fonts.googleapis.com><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700;800&display=swap" rel=stylesheet><link href="https://fonts.googleapis.com/css?family=PT+Serif:400,700" ; rel=stylesheet><link href="https://fonts.googleapis.com/css2?family=Nanum+Myeongjo" ; rel=stylesheet><link rel=stylesheet href=/css/style.min.44d270fd44b410cd7d8861ed39062788da870403b08c8580287aecbda92fc0fc.css><meta name=description content="BLAST events"><meta property="og:title" content="Events"><meta property="og:type" content="website"><meta property="og:url" content="https://jhublast.github.io/events/"><meta property="og:description" content="BLAST events"><meta name=twitter:card content="summary"></head><body class='page page-team-list'><div id=main-menu-mobile class=main-menu-mobile><ul><li class=menu-item-home><a href=/><span>Home</span></a></li><li class=menu-item-research><a href=/research/><span>Research</span></a></li><li class=menu-item-people><a href=/people/><span>People</span></a></li><li class=menu-item-events><a href=/events/><span>Events</span></a></li></ul></div><div class=wrapper><div class=header><div class=container><div class=row><div class=col-12><div class=header-inner><h3 class=header-title><a href=https://jhublast.github.io/>BLAST Working Group</a></h3></div></div></div><div id=main-menu class=main-menu><ul><li class=menu-item-home><a href=/><span>Home</span></a></li><li class=menu-item-research><a href=/research/><span>Research</span></a></li><li class=menu-item-people><a href=/people/><span>People</span></a></li><li class="menu-item-events active"><a href=/events/><span>Events</span></a></li></ul></div><button id=toggle-main-menu-mobile class="hamburger hamburger--slider" type=button aria-label="Mobile Menu">
<span class=hamburger-box><span class=hamburger-inner></span></span></button></div></div><div class=intro-small><div class=container><div class="row justify-content-start"><div class="col-12 col-md-7 col-lg-6 order-2 order-md-1"><h1 id=events>Events</h1></div></div></div></div><div class="container pb-4"><center><h2 style=font-size:2.5rem class=mb>Upcoming</h2></center><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row><div class="col-12 col-md-12 mb-2"><div class="summary summary-large"><div class=meta><h2 class=event-name>A Second Look at Spatial Confounding in Spatial Linear Mixed Models</h2><p>Tuesday, October 25, 2022</p><p>Speaker:
<a href=https://www.stat.iastate.edu/people/kori-khan>Dr. Kori Khan</a></p></div><div class=content><p>In the last two decades, considerable research has been devoted to a phenomenon known as spatial confounding. Spatial confounding is thought to occur when there is collinearity between a covariate and the random effect in a spatial regression model. This collinearity is often considered highly problematic when the inferential goal is estimating regression coefficients, and various methodologies have been proposed to &ldquo;alleviate&rdquo; it. Recently, it has become apparent that many of these methodologies are flawed, yet the field continues to expand. In this talk, we synthesize work in the field of spatial confounding. We propose that there are at least two distinct phenomena currently conflated with the term spatial confounding. We refer to these as the analysis model and the data generation types of spatial confounding. In the context of spatial linear mixed models, we show that these two issues can lead to contradicting conclusions about whether spatial confounding exists and whether methods to alleviate it will improve inference. Our results also illustrate that in many cases, traditional spatial models do help to improve inference of regression coefficients. Drawing on the insights gained, we offer a path forward for research in spatial confounding.</p></div></div></div></div></div><div class="container pb-4"><center><h2 style=font-size:2.5rem class=mb>Past</h2></center><div class=row></div><div class=row><div class="col-12 col-md-12 mb-4"><div class="summary summary-large"><div class=meta><h2 class=event-name>Efficient Alternatives for Bayesian Hypothesis Tests in Psychology</h2><p>Wednesday, October 12, 2022</p><p>Speaker:
Dr. Sandipan Pramanik</p></div><div class=content><p>Bayesian hypothesis testing procedures have gained increased acceptance in recent years. A key advantage of Bayesian tests over classical testing procedures is their potential to quantify information supporting true null hypotheses. Ironically, default implementations of Bayesian tests prevent the accumulation of strong evidence in favor of true null hypotheses because associated default alternative hypotheses assign a high probability to data that are most consistent with a null effect. We propose the use of &ldquo;non-local&rdquo; alternative hypotheses to resolve this paradox. The resulting class of Bayesian hypothesis tests permits a more rapid accumulation of evidence in favor of both true null hypotheses and alternative hypotheses that are compatible with standardized effect sizes of most interest in psychology.</p></div></div></div></div><div class=row><div class="col-12 col-md-12 mb-4"><div class="summary summary-large"><div class=meta><h2 class=event-name>Bypassing Markov Chains for Bayesian Generalized Linear Mixed Effects Models</h2><p>Wednesday, May 4, 2022</p><p>Speaker:
<a href=https://sites.google.com/site/jonathanbradley28/>Dr. Jonathan R. Bradley</a></p></div><div class=content><p>Markov chain Monte Carlo (MCMC) is an all-purpose tool that allows one to generate dependent replicates from a posterior distribution for effectively any Bayesian hierarchical model. As such, MCMC has become a standard in Bayesian statistics. However, convergence issues, tuning, and the effective sample size of the MCMC are nontrivial considerations that are often overlooked or can be difficult to assess. Moreover, these practical issues can produce a significant computational burden. This motivates us to consider finding closed-form expressions of the posterior distribution that are computationally straightforward to sample from directly. We focus on a broad class of Bayesian generalized linear mixed-effects models (GLMM) that allows one to jointly model data of different types (e.g., Gaussian, Poisson, and binomial distributed observations). Exact sampling from the posterior distribution for Bayesian GLMMs is such a difficult problem that it is now arguably overlooked as a possible problem to solve. To solve this problem, we derive a new class of distributions that gives one the flexibility to specify the prior on fixed and random effects to be any conjugate multivariate distribution. We refer to this new distribution as the generalized conjugate multivariate (GCM) distribution. The expression of the exact posterior distribution is given along with the steps to obtain direct independent simulations from the posterior distribution. These direct simulations have an efficient projection/regression form, and hence, we refer to our method as Exact Posterior Regression (EPR). Several illustrations are provided.</p></div></div></div></div><div class=row><div class="col-12 col-md-12 mb-4"><div class="summary summary-large"><div class=meta><h2 class=event-name>nnSVG: scalable identification of spatially variable genes using nearest-neighbor Gaussian processes</h2><p>Wednesday, April 20, 2022</p><p>Speaker:
<a href=https://lmweber.org/>Dr. Lukas Weber</a></p></div><div class=content><p>Spatially-resolved transcriptomics enables the measurement of transcriptome-wide gene expression along with spatial coordinates of the measurements within tissue samples. Depending on the technological platform, this is achieved either by tagging messenger RNA (mRNA) molecules with spatial barcodes followed by sequencing, or through fluorescence imaging-based in-situ transcriptomics techniques where mRNA molecules are detected along with their spatial coordinates using sequential rounds of fluorescent barcoding. During computational analyses of spatially-resolved transcriptomics data, an important initial analysis step is to apply feature selection methods to identify a set of genes that vary in expression across the tissue sample of interest. These genes are referred to as &lsquo;spatially variable genes&rsquo; (SVGs). These SVGs can then be further investigated individually as potential markers of biological processes, or used as the input for further downstream analyses such as spatially-aware unsupervised clustering of cell populations. Here, we propose &rsquo;nnSVG&rsquo;, a new scalable approach to identify SVGs based on nearest-neighbor Gaussian processes (NNGPs), which applies NNGPs in the context of spatially-resolved transcriptomics data. Our method identifies SVGs with flexible length scales per gene, optionally within spatial domains (subregions of the tissue slide), and scales linearly with the number of spatial locations. The linear computational scalability ensures that the method can be applied to the latest technological platforms with thousands or more spatial locations per tissue slide. We demonstrate the performance of our method using experimental data from several technological platforms and simulations, and show that it outperforms existing approaches. A software implementation is available from Bioconductor and GitHub.</p></div></div></div></div><div class=row><div class="col-12 col-md-12 mb-4"><div class="summary summary-large"><div class=meta><h2 class=event-name>Better with Bayes: surface-based spatial Bayesian modeling in functional MRI</h2><p>Wednesday, April 6, 2022</p><p>Speaker:
<a href=https://mandymejia.com/>Dr. Mandy Mejia</a></p></div><div class=content><p>Functional magnetic resonance imaging (fMRI) is a non-invasive indirect measure of neural activity, which is commonly used to study the function, organization and connectivity of the brain. Given its high dimensionality and complex spatiotemporal structure, fMRI data is often analyzed in a “massive univariate” framework wherein a separate model is fit at every location (e.g. voxel or vertex) of the brain. This approach ignores spatial dependencies, leading to inefficient estimates and a lack of power to detect effects, particularly in individual subjects. A statistically principled alternative is spatial Bayesian models, which impose spatial priors on the latent signal. For computational feasibility, stationary and isotropic gaussian Markov random field (GMRF) spatial priors are a common choice. The underlying signal in fMRI data is primarily localized to the gray matter of the cortical surface and subcortical/cerebellar structures. In its original volumetric form, the spatial fields of this signal exhibit clear deviations from the assumptions of stationarity and isotropy due to cortical folding and the presence of nuisance tissue classes (white matter and cerebral spinal fluid). It is therefore preferable to build spatial models directly on the cortical surface, a 2-dimensional manifold, and subcortical/cerebellar gray matter regions (collectively referred to as “grayordinates”). In this talk, I will discuss my group’s work developing spatial Bayesian models for common types of fMRI analysis. I will also discuss the software we have developed to facilitate the adoption of grayordinates neuroimaging data and spatial Bayesian modeling for fMRI. Finally, I will present an application using a task fMRI study of amyotrophic lateral sclerosis (ALS), in which we uncovered new features of disease progression.</p></div></div></div></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=row></div><div class=people-box><center><a href=/events-archive class=button>All Past Events</a></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/medium-zoom/1.0.6/medium-zoom.min.js integrity="sha512-N9IJRoc3LaP3NDoiGkcPa4gG94kapGpaA5Zq9/Dr04uf5TbLFU5q0o8AbRhLKUUlp8QFS2u7S+Yti0U7QtuZvQ==" crossorigin=anonymous referrerpolicy=no-referrer></script>
<script>const images=Array.from(document.querySelectorAll("img.research-image"));images.forEach(e=>{mediumZoom(e,{margin:0,scrollOffset:40,container:null,template:null})})</script><div class=footer><div class=container><div class=row><div class=col-12><div class=footer-inner><h3 class=footer-title>BLAST Working Group</h3><div id=footer-menu class=footer-menu><ul></ul></div></div></div></div></div></div><script type=text/javascript src=/js/scripts.min.8504133605a277da18f0d58cfd2e90d154962f4a961543a6e2f0a459a2d05462.js></script>
<script async src="https://www.googletagmanager.com/gtag/js?id=G-7VNHD4PKQE"></script>
<script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}gtag("js",new Date),gtag("config","G-7VNHD4PKQE")</script></body></html>